Source/SwiftLintFramework/Rules/FirebaseConfigDefaultsRule.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  FirebaseConfigDefaultsRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigDefaultsRule: ASTRule, RecursiveRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_defaults",
+        name: "Firebase Config Defaults",
+        description: "Firebase Config defaults should be set.",
+        nonTriggeringExamples: [
+            "class ViewController: UIViewController {\n" +
```

Comment:
Consider adding more examples that do not trigger the violation. Like a `viewDidLoad()` implementation that doesn't have the `.fetch(withExpirationDuration:)` inside.